hms-app/src/actions/HackathonActions.tsx

```diff
@@ -43,13 +43,15 @@ export const createHackathon = async (
   instance: IPublicClientApplication,
   title: string,
   DescriptionValue: string,
+  slug: string,
   startDate: Date,
   endDate: Date
 ) => {
   const idToken = await getIdToken(instance)
   const options = buildFetchOptions('POST', idToken, {
     title,
     description: DescriptionValue,
+    slug,
     startDate: startDate.toString(),
     endDate: endDate.toString(),
```

Comment:
when you build a type like this it's important to name the parameters for clarity and safety
```suggestion
    title: title,
    description: DescriptionValue,
    slug: slug,
    startDate: startDate.toString(),
    endDate: endDate.toString(),
```